ui/src/components/JsonDisplay/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { string, object } from 'prop-types';
+import { dump } from 'js-yaml';
+import { Switch, Typography, Grid } from '@material-ui/core';
+import Code from '../Code';
+import 'highlight.js/styles/atom-one-dark.css';
+
+const JsonDisplay = props => {
+  const YamlJsonSwitch = ({ handleChange, value }) => {
```

Comment:
Fat arrow function has an implicit return, so if you're not doing anything before the return block on line 10 you can remove it and the brackets.

Reply:
Yeah I know just usually I add some code to it, than forget to remove return :D